ciphers/uint128_t.hpp

```diff
@@ -0,0 +1,1075 @@
+/**
+ * @file uint128_t.hpp
+ *
+ * @details Implementation of 128-bit unsigned integers.
+ * @note The implementation can be flagged as not completed. This header is used
+ * with enough operations as a part of bigger integer types 256-bit integer.
+ * @author [Ashish Daulatabad](https://github.com/AshishYUO)
+ */
+
+#include <algorithm>
+#include <ostream>
+#include <string>
+#include <utility>
```

Comment:
Add a one-line description of what the library/header is for (see the example below).
```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```